### DISCUSSION TITLE
Icons

### DISCUSSION THREAD

**Discussion by boxadesign** [ONSdigital/design-system#95] [Feedback]
Use this place to discuss [icons](https://ons-design-system.netlify.com/styles/typography#icons) in the [ONS Design System](https://ons-design-system.netlify.com)

**Comment by jrbarnes9** [comment]
Some icons were hidden when screen colours inverted (using v12 ZoomFusion) 
Breadcrumbs: #454 Details: #453

**Comment by jrbarnes9** [comment]
I think we should probably use a single icon for directional calls to action.

So I propose we update the icons used for a cta button, back link, and single breadcrumb to the same icons used in the content page (`arrow-next` and `arrow-previous`)

![Screenshot 2021-10-27 at 10 40 32](https://user-images.githubusercontent.com/43346934/139416199-e5e72877-cca7-4c54-8459-0b709d50e067.png)
![Screenshot 2021-10-29 at 10 57 14](https://user-images.githubusercontent.com/43346934/139416204-4a010e9e-0f5c-4944-afaf-b2c8ef8f78f6.png)


...also consider adding the icons to the next/prev pagination links...
![Screenshot 2021-10-29 at 11 20 28](https://user-images.githubusercontent.com/43346934/139418926-28707b86-d7ea-40fb-b8d4-45cdcd4d797e.png)
